consensus-types/epbs/indexed_payload_attestation_test.go

```diff
@@ -0,0 +1,60 @@
+package epbs
+
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	eth "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/testing/require"
+)
+
+func TestGetAttestingIndices(t *testing.T) {
+	attestingIndices := []primitives.ValidatorIndex{1, 2, 3}
+	nilAttestingIndices := []primitives.ValidatorIndex{}
```

Comment:
This is an empty slice not nil slice. It's a non-nil and empty slice. Calling it 'nil' is confusing at least to me.

Reply:
If you really want a nil slice, you can do `nilAttestingIndices := []primitives.ValidatorIndex(nil)`